eth/db/journal.py

```diff
@@ -24,237 +26,230 @@ class DeletedEntry:
 # 1. key in wrapped
 # 2. key modified in journal
 # 3. key deleted
-DELETED_ENTRY = DeletedEntry()
+DELETE_WRAPPED = DeletedEntry()
 
 # 1. key not in wrapped
 # 2. key created in journal
 # 3. key deleted
-ERASE_CREATED_ENTRY = DeletedEntry()
+REVERT_TO_WRAPPED = DeletedEntry()
+
+ChangesetValue = Union[bytes, DeletedEntry]
+ChangesetDict = Dict[bytes, ChangesetValue]
+
+id_generator = count()
```

Comment:
Maybe `get_checkpoint_id = count().__next__`?  Might look a little more natural in the places where it's used.